scripts/create_baseline_stubs.py

```diff
@@ -65,14 +65,9 @@ def run_black(stub_dir: str) -> None:
     subprocess.run(["black", stub_dir])
 
 
-def run_isort(stub_dir: str) -> None:
-    print(f"Running isort: isort {stub_dir}")
-    subprocess.run([sys.executable, "-m", "isort", stub_dir])
-
-
 def run_ruff(stub_dir: str) -> None:
     print(f"Running Ruff: ruff {stub_dir}")
-    subprocess.run([sys.executable, "-m", "ruff", stub_dir])
+    subprocess.run([sys.executable, "-m", "ruff", stub_dir, "--fix-only"])
```

Comment:
What does this do?

Reply:
Fix errors it can fix, and don't report on things it can't.

Ruff now recommends explicitly adding `check`:
```suggestion
    subprocess.run([sys.executable, "-m", "ruff", "check", stub_dir, "--fix-only"])
```

Reply:
> Ruff now recommends explicitly adding check

Right because of `ruff format` and to be explicit. I'll update other code locations too.